snuba/clickhouse/formatter/expression.py

```diff
@@ -202,7 +202,12 @@ def _format_date_literal(self, exp: Literal) -> str:
         )
 
 
-class ExpressionFormatterAnonymized(ExpressionFormatterBase):
+def _gen_random_number() -> int:
+    # https://xkcd.com/221/
+    return -1337
```

Comment:
Both changes you've commented on are already in the codebase and were introduced by @vladkluev in this PR:
https://github.com/getsentry/snuba/pull/2308

I believe the single quotes are meant to show that it is indeed a string, and the number to show it is a number instead of `$S` and `$N`?

Reply:
yes, this way you can copy and paste the query to run/profile it without needing to find and replace in it.

Reply:
Gotcha, that makes sense.